plugin/src/main/resources/com/redhat/jenkins/plugins/ci/form/dropdownInstanceSelector.jelly

```diff
@@ -60,7 +60,7 @@ THE SOFTWARE.
           <f:invisibleEntry> <!-- Inject provider name so all config fragments do not have to repeat that -->
             <input type="hidden" name="name" value="${pdata.name}"/>
           </f:invisibleEntry>
-          <j:set var="instance" value="${instance.providerData}"/>
+          <!-- <j:set var="instance" value="${instance.providerData}"/> -->
```

Comment:
Should this just be removed?  It's being set somewhere else, right?

Reply:
removing